### PR TITLE
OpenMPI and CI

### PR DESCRIPTION
**Pull request by branfosj**

### CODE REVIEW

**Comment by branfosj** [comment]
Sadly CUDA and Python fail to install on power9.

File: test/easyconfigs/easyconfigs.py [hunk 1]
```diff
@@ -64,6 +64,12 @@
 # other than optimizing for time, this also helps to get around problems like http://bugs.python.org/issue10949
 single_tests_ok = True
 
+# Exclude these tool chains from tests
+EXCLUDE_TOOLCHAINS = ['{}-{}'.format(x, y) for x in ['foss', 'intel', 'fosscuda', 'intelcuda', 'iomkl']
+                                           for y in ['2014', '2015', '2016', '2017', '2018']]
+EXCLUDE_TOOLCHAINS += ['{}-{}'.format(x, y) for x in ['GCC', 'GCCcore'] for y in ['4.', '5.', '6.', '7.']]
+EXCLUDE_TOOLCHAINS += ['ictce', 'gimkl', 'giolf', 'golf', 'goolf']
+
```
Comment (orbsmiv) [inline test/easyconfigs/easyconfigs.py]: Minor style point but should we be using `EXCLUDE_TOOLCHAINS.extend()` instead of `+=`?

**Comment by branfosj** [comment]
I think we should leave the Python 2 -> 3 for upstream Easybuild. I'd prefer to stick with whatever they do and not go off on our own.

**Comment by branfosj** [comment]
CUDA problem is https://devtalk.nvidia.com/default/topic/1063995/cuda-setup-and-installation/cuda-10-1-243-10-1-update-2-ppc64le-run-file-installation-issue/

We'll need to decide what to do...

**Comment by branfosj** [comment]
CUDA 10.1.105 does install. CUDA 10.1.168 does not.

File: test/easyconfigs/easyconfigs.py (hunk 1 above)
Comment (branfosj) [reply test/easyconfigs/easyconfigs.py]: done

**Comment by branfosj** [comment]
Remembered that @edmondac had fixed the `pthread` issue on `ppc64le` again in the past and I just had to dig it up and find out where to put it. Adding this does not cause any issues for building on Intel, so I suggest we should just have it in for all.

**Comment by branfosj** [comment]
My testing has been `Python-3.7.4-GCCcore-8.3.0.eb`, `foss-2019b.eb` and `fosscuda-2019b` (where appropriate).

**Pull request merged**